asn1rs-model/src/generate/rust.rs

```diff
@@ -90,6 +93,18 @@ impl RustCodeGenerator {
         self
     }
 
+    pub fn add_local_attr<N: Into<String>, I: Into<String>>(&mut self, name: N, attr: I) {
+        self.local_attrs
+            .entry(name.into())
+            .or_default()
+            .push(attr.into());
```

Comment:
Why not just a `self.local_attrs.push(name.into(), attr.into());`?

Reply:
This collects attributes for the same name into a vec, so that we have (name, [attr...]) instead of  [(name, attr)...]. Meaning we don't need to store the "name" key for each entry.

However, for this use case probably doesn't matter too much. Do you still want me to change it?

Reply:
Ah, my bad. This is fine :)